File: Iniciante/1045.py
V = input().split()
V = list(map(float,V))

A, B, C = sorted(V)[::-1]

con = True

if(A >= B+C):
    print('NAO FORMA TRIANGULO')
    con = False

if(A**2 == (B**2) + (C**2) and con):
    print('TRIANGULO RETANGULO')
if(A**2 > (B**2) + (C**2) and con):
    print('TRIANGULO OBTUSANGULO')
if(A**2 < (B**2) + (C**2) and con):
    print('TRIANGULO ACUTANGULO')
if(A == B and B == C and con):
    print('TRIANGULO EQUILATERO')
if((A == B or B == C) and not (A == B and B == C) and con):
    print('TRIANGULO ISOSCELES')

File: Iniciante/1045.php
<?php

$L1 = explode(" ", readline());
rsort($L1);

if($L1[0] >= $L1[1] + $L1[2]){
    printf("NAO FORMA TRIANGULO\n");

}elseif($L1[0]**2 == $L1[1]**2 + $L1[2]**2){
    printf("TRIANGULO RETANGULO\n");

}elseif($L1[0]**2 > $L1[1]**2 + $L1[2]**2){
    printf("TRIANGULO OBTUSANGULO\n");

}else{
    printf("TRIANGULO ACUTANGULO\n");

}
if($L1[0] == $L1[1] && $L1[1] == $L1[2]){
    printf("TRIANGULO EQUILATERO\n");

}elseif($L1[0] == $L1[1] || $L1[1] == $L1[2]){
    printf("TRIANGULO ISOSCELES\n");

}

?>